src/routers/item.js

```diff
@@ -10,8 +10,8 @@ router.get('/available', itemController.getAvailableItems);
 // GET route for /api/items/filter
 router.get('/filter', itemController.getFilteredItems);
 
-// POST route for /api/items
-router.post('/', itemController.addItem);
+// POST route for /api/items/item
```

Comment:
@AmmarBaki2 there is no need for api/items/item, it doesn't make sense, don't you think? Let's keep api/items. We need to revise sigma instead.

Reply:
Thanks for pointing out 😁 
I looked at previous assignments and it is done like you said 👍